### ISSUE TITLE
Variable in "dir" breaks shell expanded variables

### ISSUE THREAD

**Issue by ahawkins** [go-task/task#426]
- Task version: 3.2.0 (working on 3.1.0)
- Operating System: MacOS

### Example Taskfile showing the issue

I think this is related to #384 

```yaml
version: "3"

vars:
  PROJECT_ROOT:
    sh: echo "Dummy"

tasks:
  init:
    dir: test/{{ .PROJECT }}
    cmds:
      - echo '{{ .PROJECT_ROOT }}'
```

Break it:

```
$ mkdir -p test/foo
$ task init PROJECT=foo
task: Command "echo "Dummy"" in taskvars file failed: could not stat: stat /Users/ahawkins/src/skillshare/scratch/test/{{ .PROJECT }}: no such file or directory
```

I expect that `{{ .PROJECT }}` should be interpolated.

**Comment by ahawkins** [comment]
Adding @rvangundy

**Comment by andreynering** [comment]
Hi @ahawkins, thanks for reporting!

This was indeed a regression, and I just released a fix in [v3.2.1](https://github.com/go-task/task/releases/tag/v3.2.1).

**Comment by ahawkins** [comment]
Awesome. Thank you for all your work on Task <3.

// Adam
